content/en/docs/setup/_index.md

```diff
@@ -97,7 +97,7 @@ The following production environment solutions table lists the providers and the
 | [OpenShift](https://www.openshift.com)          |[OpenShift Dedicated](https://www.openshift.com/products/dedicated/) and [OpenShift Online](https://www.openshift.com/products/online/)              |        | [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)            |      |  [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)        |[OpenShift Container Platform](https://www.openshift.com/products/container-platform/)
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
 | [oVirt](https://www.ovirt.org/)         |              |        |             |     | &#x2714;          |
-| [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service)         |              | &#x2714;       | &#x2714;           |      |          |
+| [Pivotal](https://pivotal.io/) | | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service) | [Enterprise Pivotal Container Service (PKS)](https://pivotal.io/platform/pivotal-container-service)           |      |          |
```

Comment:
Should this page also have links to Cloud PKS and Essential PKS?

https://content.pivotal.io/blog/meet-the-pks-family-flexibility-for-your-kubernetes-architectures says:
> All three products are available from Pivotal and VMware.

Reply:
I didn't see a `pivotal.io` landing page for the other two products. On the page you linked, the Cloud PKS and Essential PKS links go to VMware.

Reply:
How about: list those things, but don't make them be hyperlinks?
(none of the options feels ideal, I'm trying to recommend a good compromise)